### PR TITLE
Fix delete translation for Django 1.9+

### LINKED ISSUE #122: delete-translation URL broken?
Hi all,

When I try to delete a translation in the admin I get a 404. Turns out the URL:
/en/admin/events/event/2/change/delete-translation/en/
is wrong.
When I remove the "/change" from it, it works:
/en/admin/events/event/2/delete-translation/en/

Is this a bug?

I am running 
Django==1.9.2
django-parler==1.6.1
django-cms==3.2.1

### PR DESCRIPTION
**Pull request by yakky**
Fix #122 by registering the appropriate url path according to the django version

### CODE REVIEW

File: parler/admin.py [hunk 1]
```diff
@@ -400,6 +408,7 @@ def delete_translation(self, request, object_id, language_code):
 
         # Get object and translation
         shared_obj = self.get_object(request, unquote(object_id))
+        print(object_id)
```
Comment (vdboor) [inline parler/admin.py]: Could you remove this one? ;-)

Comment (yakky) [inline parler/admin.py]: Argh! :D

Comment (yakky) [inline parler/admin.py]: Done

**Comment by vdboor** [comment]
Thanks!

**Pull request merged**